vtrace/platforms/gdbstub.py

```diff
@@ -846,24 +932,63 @@ def getMemoryMaps(self, pgsize=4096):
                         # of the next
                         off += pgsize
                         data = self.gdbReadMem(off + qoff - 5, 10)
-                        print("DEBUG: %x:  %r" % (off+qoff-5, data))
+                        logger.debug('%#x-%#x: %r', off+qoff-5, off+qoff+5, data)
 
                 except Exception as e:
-                    print("DEBUG: %r" % e)
+                    logger.exception('failed to read %#x-%#x from gdb', off+qoff-5, off+qoff+5)
 
                 maps.append((qoff, off, 7, b'offmap-%s' % offname))
                 # TODO: scour large amount of address space looking for maps?
 
         return maps
 
-            
-    def gdbGetFeatureFile(self, fname=b'target.xml'):
-        return self.qXfer(b'features', name=fname)
-
+    def gdbGetXMLFile(self, fname=b'target.xml'):
```

Comment:
wait, you changed the name of this function, and it's functionality and i don't see a similar change in `gdb_client.py`
please make sure these changes are supported by the rest of the codebase.

Reply:
I am not sure why I changed the name of this function unnecessarily. I will change it back.